README.md

```diff
@@ -194,10 +197,9 @@ Shell aliases and scripts:
 * `b` for `bundle`.
 * `g` with no arguments is `git status` and with arguments acts like `git`.
 * `git-churn` to show churn for the files changed in the branch.
-* `m` for `rake db:migrate && rake db:rollback && rake db:migrate && rake db:test:prepare`.
+* `migrate` for `rake db:migrate && rake db:rollback && rake db:migrate && rake db:test:prepare`.
```

Comment:
While we're in here, [this alias][1] is actually `rake db:migrate db:rollback && rake db:migrate`

[1]: https://github.com/thoughtbot/dotfiles/blob/74fd8a362ea295ff4bba8479e09846e8abe5f939/aliases#L12